Docs: Add Privy Docs to Quickstart and DevTools (#316)
rsksmart/devportal: tutorials/resolve-nifty-issue.md

---
layout: rsk
title: Resolve Nifty Wallet Issue

> description: "Resolution for breaking change in derivation path for Nifty Wallet 5.1.0"

Reviewer: ```suggestion
description: "Resolution for breaking change in derivation path for Nifty Wallet 5"
```

description: "Resolution for breaking change in derivation path for Nifty Wallet 5.1.0"
tags: tutorial, nifty, chainId, address, derivation, bip44, slip44
---

## Am I affected?

First, check your Nifty wallet version:
- Click the menu on the right (three horizontal lines)
- Click "Info/Help"

> - If version is "5.0.0", then read the next section. If not, you are OK.

Reviewer: Not sure why we're removing these instructions,
but I guess it is OK, the user is likely to be able to figure this out using Nifty's own docs anyway.

> ## Nifty wallet version 5.1.0

Reviewer: ```suggestion
## Nifty wallet version 5
```

Changing this to only specify the major version here and in the title,
since this affects multiple minor versions.
Stating `5.1.0` would imply that `5.1.0` is affected,
but `5.0.0` is not, but this is shortly contradicted by the release notes links below.
Just removing the potential for confusion here.

This was a **breaking change**.
Check out Nifty's release notes for more information:
[Nifty Wallet Release 5.0.0](https://forum.poa.network/t/nifty-wallet-release-5-0-0/3335)
and [Nifty Wallet Release 5.1.0](https://forum.poa.network/t/nifty-wallet-release-5-1-0/3440).

Also check out RSK's documentation on our
[derivation paths](/rsk/architecture/account-based/ "Account Based RSK Addresses").

## If you created your wallet after this release

Continue using Nifty Wallet.
No further action is necessary.

## If you created your wallet before this release

> If you created your wallet (and account) prior to this release,
> you will see this label: "The account is derived from
> ETH derivation path despite you connected to another chain.
> If you are ready to switch to correct derivation path, just
> restore from the same seed phrase.".
> In other words, this means that your seed phrase was used to
> generate your account as if it were an ETH account and not an
> RSK account.
> You will see your account as usual until you restore your seed.
> Once you do that, you won't find your former account and,
> therefore, your balance.
> If that happen and you didn't move your funds, not to worry,
> you will be able to derive the **original set** of addresses by
> [adding RSK as a custom node](#add-rsk-as-custom-node).

Reviewer: ```suggestion
If you created your wallet (and therefore accounts)
prior to this release, you will see this label:

> The account is derived from
> ETH derivation path despite you connected to another chain.
> If you are ready to switch to correct derivation path, just
> restore from the same seed phrase.

In other words, this means that your seed phrase was used to
generate your account as if it were an Ethereum account
and not an RSK account.
You will see your account as usual until you restore your seed.
Once you do that, you won't find your former account and,
therefore, your balance.
If that happens, and you didn't move your funds, not to worry,
you will be able to derive the **original set** of addresses by
[adding RSK as a custom node](#add-rsk-as-custom-node).
```
